cts/settings/base.py

```diff
@@ -274,15 +274,19 @@
     PROJECT_ROOT, 'shipments', 'data', 'TM_WORLD_BORDERS_SIMPL-0.3.shp'
 )
 
-
+# We set `expires` so that if the queue gets hung or the workers go down,
+# tasks don't pile up in the queue leading to a thundering herd problem
+# when things start running again.
 CELERYBEAT_SCHEDULE = {
     'process_new_scans': {
         'task': 'ona.tasks.process_new_scans',
         'schedule': timedelta(minutes=15),
+        'expires': 10*60,  # 10 minutes
```

Comment:
[This](http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#available-fields) makes it look like `expires` should be inside an `options` dictionary.